### PR TITLE
[move-prover][move language] Cleaned up comments for RegisteredCurren…

### PR DESCRIPTION
**Pull request by DavidLDill**
…cies.

Tidied up and reorganized comments.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

**Review by wrwg — APPROVED** [review]
I saw also changes to RegisteredCurrencies in your Libra PR (or maybe not?) Just to ensure that is intentionally.

File: language/stdlib/modules/RegisteredCurrencies.move [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+/// string (vector<u8>) for the currency name to vector of names in LibraConfig.
 module RegisteredCurrencies {
     use 0x1::Errors;
     use 0x1::LibraConfig;
     use 0x1::LibraTimestamp;
     use 0x1::Roles;
     use 0x1::Vector;
 
-    /// An on-chain config holding all of the currency codes for registered
+    /// An LibraConfig config holding all of the currency codes for registered
```
Comment (emmazzz) [inline language/stdlib/modules/RegisteredCurrencies.move:12]: Change "An" to "A"?

**Pull request closed without merging**